velox/docs/functions/spark/string.rst

```diff
@@ -20,6 +20,21 @@ Unless specified otherwise, all functions return NULL if at least one of the arg
     If ``n < 0``, the result is an empty string.
     If ``n >= 256``, the result is equivalent to chr(``n % 256``).
 
+.. spark:function:: concat_ws(separator, [string]/[array<string>], ...) -> varchar
+
+   Returns the concatenation for ``string`` & all elements in ``array<string>``, separated by
+   ``separator``. Only accepts constant ``separator``. It takes variable number of remaining
+   arguments. And ``string`` & ``array<string>`` can be used in combination.  If ``separator``
+   is NULL, returns NULL, regardless of the following inputs. If only ``separator`` (not a
+   NULL) is provided or all remaining inputs are NULL, returns an empty string. ::
```

Comment:
Would you clarify the behavior when the separator is not NULL, but string or array(string) contain NULL? Maybe also include an example.

Reply:
Just clarified and added an example.